### DISCUSSION TITLE
Asus B150m-plus rx580 can not work,need some suggestion. please

### DISCUSSION THREAD

**Discussion by iknightpl** [xCuri0/ReBarUEFI#172] [General]
Hey Guys, can anybody help me to check If any steps I do has mistakes?
My motherboard is asus b150m-plus. RX580 2048sp
I have followed steps like above 5 steps shows in image.
there is some messages I don't understand on the image.
after that I flash it to my motherboard and my pc can boot.
then disable csm, open above 4g decoding.
But when I had set reBarState to 10(1G), after reboot, the system halt with information on screen: b1Initializelibrary failed!
Thanks everyone.
oh my card is: yeston rx580 2048sp .
and in the regedit, I have 0000,0001,0002 entry. this card is 0002. I have change the reg file from 0000 to 0002.

![1](https://github.com/xCuri0/ReBarUEFI/assets/52777278/9b28ec91-cf7f-4ca1-9cf2-cdab52e7675a)
![2](https://github.com/xCuri0/ReBarUEFI/assets/52777278/d108c8d1-4103-435c-855e-03394855f70e)
![3](https://github.com/xCuri0/ReBarUEFI/assets/52777278/bbde0baa-d5f9-4ca8-b89a-bc3605a4aa57)
![4](https://github.com/xCuri0/ReBarUEFI/assets/52777278/6119dc92-7c26-4f3f-8e69-afab344f7e1f)
![5](https://github.com/xCuri0/ReBarUEFI/assets/52777278/8aa616e7-6a68-40cc-bee4-4c2769fd9eb7)

**Comment by xCuri0** [comment]
try larger size like 8gb ?

**Reply by iknightpl** [reply]
Ah, is that the problem? I see that other discussions are say that starting from a lower size, so I just try 1G and did not try 8G. I will try it and provide feedback. Thank you for your reply.

**Reply by xCuri0** [reply]
yes its most likely the cause

**Reply by iknightpl** [reply]
Oh , It works property!
That is the reason.
Thanks!
